tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,187 @@
+
+    In Godot 4.3 we have implemented a unified approach that is explained on this help page
+    so you don't need to worry about these differences, the :ref:`XRInterface <class_xrinterface>`
+    implementation is now responsible for applying the correct platform dependent method [#]_.
+
+    For headsets such as the Meta Quest and HTC Elite you will need to use the
+    `OpenXR vendors plugin v3.0.0 <https://github.com/GodotVR/godot_openxr_vendors/releases>`__
+    or later to enable passthrough.
+
+    For backwards compatibility the old API for passthrough is still available but it is recommended
+    to follow the new instructions below.
+
+Environment blend modes
+-----------------------
+
+The way we configure VR or AR functionality is through setting the environment blend mode.
+This mode determines how the (real world) environment is blended with the virtual world.
+
+.. list-table:: Blend modes
+  :widths: 35 65
+  :header-rows: 1
+
+  * - Blend mode
+    - Description
+  * - XR_ENV_BLEND_MODE_OPAQUE
+    - The rendered image is opaque, we do not see the real world. We're in VR mode.
+      This will turn off passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ADDITIVE
+    - The rendered image is added to the real world. This mode generally applies to AR only devices
+      such as the Hololens where the optics are not able to obscure the real world.
+      This will turn on passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ALPHA_BLEND
+    - The rendered image is alpha blended with the real world.
+      This is supported on devices that can fully obscure the real world
+      and generally the preferred way to do AR.
+      This will turn on passthrough if passthrough is used.
+  
+You can set the environment blend mode for your application through the ``environment_blend_mode``
+property of the :ref:`XRInterface <class_xrinterface>` instance.
+
+You can query the supported blend modes on the hardware using the
+``get_supported_environment_blend_modes`` property on the same instance.
+
+Configuring your background
+---------------------------
+
+When setting the blend mode to ``XR_ENV_BLEND_MODE_ALPHA_BLEND`` you must set
+the ``transparent_bg`` property on :ref:`Viewport <class_viewport>` to true.
+It is advisable to do the same with ``XR_ENV_BLEND_MODE_ADDITIVE`` however here you
+can alternatively set your background color to black.
+
+Either solution will result in the background rendering not contributing to lighting.
+It is thus also recommended you adjust your environment settings accordingly and ensure
+there is adequate ambient light set to illuminate your scene.
+
+.. note::
+
+    Some AR SDKs do provide ambient lighting information or even provide a full radiance
+    map to allow for real world reflections in your virtual objects.
+    The core Godot XR functionality doesn't currently have support for this, however this
+    functionality can be exposed through plugins.
```

Comment:
Note sure if this is relevant to this documentation.

Reply:
As more and more vendors are working on this, and it is a question I've been asked a few times, I think it is handy to have a reference to this. But maybe it's overkill :)